src/main/java/edu/cornell/kfs/module/cg/document/CuAwardMaintainableImpl.java

```diff
@@ -20,9 +17,10 @@ protected void refreshAward(boolean refreshFromLookup) {
         Proposal tempProposal = getAward().getProposal();
         award.setProposal(tempProposal);
         
-        getNewCollectionLine(AWARD_SUBCONTRACTORS).refreshNonUpdateableReferences();
-        getNewCollectionLine(AWARD_PROJECT_DIRECTORS).refreshNonUpdateableReferences();
-        getNewCollectionLine(AWARD_ACCOUNTS).refreshNonUpdateableReferences();
+        getNewCollectionLine(KFSPropertyConstants.AWARD_SUBCONTRACTORS).refreshNonUpdateableReferences();
+        getNewCollectionLine(KFSPropertyConstants.AWARD_PROJECT_DIRECTORS).refreshNonUpdateableReferences();
+        getNewCollectionLine(KFSPropertyConstants.AWARD_FUND_MANAGERS).refreshNonUpdateableReferences();
```

Comment:
Can you please check whether we were purposely excluding the refresh of the AWARD_FUND_MANAGERS in our customization? I see that our old cu-kfs code does not refresh the fund managers collection.

Reply:
This change seems to have gotten lost somehow. I think we need to keep it. It does not look like it is what we have customized locally.

Reply:
Ok, I'm fine with keeping this specific change. Thanks for checking.